actionpack/test/controller/filters_test.rb

```diff
@@ -15,6 +15,7 @@ def before_actions
   end
 end
 
+# <tt>filters</tt> refers to action callbacks, previously being call 'filters'.
```

Comment:
@rafaelfranca you mean that I should omit the comment and make a followup PR or omit the comment and change the file from filters_test.rb to callbacks_test.rb ?